solara/components/switch.py

```diff
@@ -0,0 +1,55 @@
+from typing import Callable, Union
+
+import reacton.ipyvuetify as v
+
+import solara
+
+
+@solara.value_component(bool)
+def Switch(
+    *,
+    label=None,
+    value: Union[bool, solara.Reactive[bool]] = True,
+    on_value: Callable[[bool], None] = None,
+    disabled=False,
+    style: str = None,
```

Comment:
See bc81d7d93a608438674118afb0f27183de956afd for how we prefer the style argument

Please also add classes.